Tools/autotest/arducopter.py

```diff
@@ -8492,6 +8492,21 @@ def AUTO_LAND_TO_BRAKE(self):
         self.change_mode('AUTO')
         self.wait_disarmed()
 
+    def mav_landed_state_takeoff(self):
+        self.set_message_rate_hz(id=mavutil.mavlink.MAVLINK_MSG_ID_EXTENDED_SYS_STATE, rate_hz=1)
```

Comment:
This is not something that the suite will revert, unfortunately.

We'll need to do try/catch and revert the message-rate-set in the case we get an exception (and at the end of the test).  If you search for `set_message_rate_hz` you should find an example.

I probably should add reverting of message rates.